src/core/lib/surface/call.h

```diff
@@ -265,6 +265,16 @@ void grpc_call_log_batch(const char* file, int line, const grpc_op* ops,
 
 void grpc_call_tracer_set(grpc_call* call, grpc_core::ClientCallTracer* tracer);
 
+// Both sets and manages call tracer life cycle.
```

Comment:
Replace with - Sets call tracer on the call and manages its life by using the call's arena.